.changeset/rare-parents-pretend.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/create-app': patch
+---
+
+Accept `PermissionEvaluator` together with the deprecated `PermissionAuthorizer`.
+
+Apply the following to `packages/backend/src/types.ts`:
+
+```diff
+- import { ServerPermissionClient } from '@backstage/plugin-permission-node';
++ import {
++  PermissionAuthorizer,
++  PermissionEvaluator,
++ } from '@backstage/plugin-permission-common';
+
+  export type PluginEnvironment = {
+    ...
+    discovery: PluginEndpointDiscovery;
+    tokenManager: TokenManager;
+    scheduler: PluginTaskScheduler;
+-   permissions: ServerPermissionClient;
++   permissions: PermissionEvaluator | PermissionAuthorizer;
```

Comment:
Just mentioning briefly that this means we'll ask the user to perform one more migration in the future when the old type disappears. I see that the actual skeleton only has `PermissionEvaluator` here so I presume it's an interim thing.